thefuzz/process.py

```diff
@@ -23,11 +24,14 @@ def _get_processor(processor, scorer):
                       fuzz.UWRatio, fuzz.UQRatio):
         return processor
 
-    if not processor:
-        return utils.full_process
+    force_ascii = scorer not in [fuzz.UWRatio, fuzz.UQRatio]
+    pre_processor = partial(utils.full_process, force_ascii=force_ascii)
+
+    if not processor or processor == utils.full_process:
```

Comment:
This is basically the same as in the original implementation now even though it is not actually true. This is an optimization to ensure that the preprocessor function is only run once when both of them are `utils.full_process`. However for most scorers this is actually `utils.full_process(utils.full_process(string), force_ascii=True)`, which doesn't behave exactly the same as `utils.full_process(string, force_ascii=True)` when used with types that are not strings (see #57).

Still this restores the previous behavior.